### DISCUSSION TITLE
Loco provides web service and can schedule schedulers at the same time?

### DISCUSSION THREAD

**Discussion by tsailin-bit** [loco-rs/loco#1088] [Q&A] [answered]
Why doesn't the scheduler start when starting with cargo loco start? You also cannot configure the configuration to enable scheduling when starting a service. In the future, when an application provides a web service, it can decide whether to start a scheduling task based on the configuration?

**Comment by wdcodecn** [comment]
you need run ```cargo loco scheduler ```

**Comment by BWStearns** [comment]
~This does seem unintuitive. My expectation was that the scheduler would start the same way that the loco workers would start. Would there be any objection to adding this?~

Just saw that this was added in 0.15.0!

**Comment by kaplanelad** [comment, marked as answer]
Use `cargo loco start --all`
